### PR TITLE
Prepare changelog

### PR DESCRIPTION
**Pull request by MaxTru**
* Adds changelog
* Gets rid of promise chain

### CODE REVIEW

File: packages/dmn-js-shared/src/base/Manager.js [hunk 1]
```diff
@@ -169,6 +169,11 @@ export default class Manager {
         return reject(parseError);
       }).then((result) => {
 
+        // break promise chain if we previously rejected or options.open was falsy
+        if (!result) {
```
Comment (pinussilvestrus) [inline packages/dmn-js-shared/src/base/Manager.js:173]: Do we need a test case for this?

File: packages/dmn-js/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,18 @@ All notable changes to [dmn-js](https://github.com/bpmn-io/dmn-js) are documente
 
 ___Note:__ Yet to be released changes appear here._
 
+## 11.0.0
+
+* `FEAT`: make `#fromXML`, `#saveXML`, `#saveSVG`, and `#open` APIs awaitable ([#514](https://github.com/bpmn-io/dmn-js/issues/514))
+* `FIX`: correct event life-cycle of `#importXML` API for error case ([`49fcb1b`](https://github.com/bpmn-io/dmn-js/commit/49fcb1b986aa95bb7fce9935029f62bc7151ee90))
+* `FIX`: fix typo in import.done event property `warnings` ([`4ef46e0`](https://github.com/bpmn-io/dmn-js/commit/4ef46e073f45e0bd2d09c4ea0d9cb21d4555318c))
```
Comment (pinussilvestrus) [inline packages/dmn-js/CHANGELOG.md:13]:
```suggestion
* `FIX`: fix typo in `import.done` event property `warnings` ([`4ef46e0`](https://github.com/bpmn-io/dmn-js/commit/4ef46e073f45e0bd2d09c4ea0d9cb21d4555318c))
```

File: packages/dmn-js-shared/src/base/Manager.js (hunk 1 above)
Comment (MaxTru) [reply packages/dmn-js-shared/src/base/Manager.js]: I discussed with Maciej -> we came to the conclusion to not do that but to simply avoid multipe chained `.then`s. This way we cannot get in the situation, that I tried to work around with this check.

Comment (MaxTru) [reply packages/dmn-js-shared/src/base/Manager.js]: Commit got replaced by a simplification of the promise chain.

Comment (MaxTru) [reply packages/dmn-js-shared/src/base/Manager.js]: BTW: intersting hour-of-code topic - I will add it for next week

File: packages/dmn-js/CHANGELOG.md (hunk 2 above)
Comment (MaxTru) [reply packages/dmn-js/CHANGELOG.md]: done :+1: 
thanks

**Review by pinussilvestrus — APPROVED** [review]
The promise chain still looks a bit messy, but I like the initiative to improve it + discuss it in the next Hour of Code. 

Changelog looks good to me 👍

**Pull request merged**